### ISSUE TITLE
Effective teaching in R

### ISSUE THREAD

**Issue by tracykteal** [ropensci/unconf#23]
At the unconference will be a lot of people involved in various teaching efforts in R. It seems like a great opportunity to get together to discuss what has been working most effectively and in different formats (workshops, courses, MOOCs) and where we might share or help contribute to each other's materials. 

@jennybc @karthik @eddelbuettel @mine-cetinkaya-rundel @richfitz @benmarwick @rdpeng @fmichonneau @amyehodge @hadley 

For Data Carpentry, I'm interested in how can we can teach people new to R the big wins, and the things that they'll really use, without presenting things as too magical. We're also interested in content and perspectives on teaching how to work with APIs in R.

**Comment by jhollist** [comment]
We have a growing group of R Users at USEPA and I have been doing some training lately trying to help build R skills within our Office of Research and Development.  So I would be really interested in what comes out of this discussion.   I will, however, not be there.  Don't know if there is a good way to listen in on these discussions, but I would love to do that if possible.  Can, at a minimum, follow along with issues on github.

**Comment by jordansread** [comment]
Triple upvote on this one. We at USGS are also using @jhollist 's materials (too bad he won't be there) and hope to undergo the data carpentry training in the near future. We are targeting R skill building across the Bureau. Would love to contribute to this session and learn from the rest of you.

**Comment by eddelbuettel** [comment]
Good to discuss in person.  I think the email discussion over at 'pentry between the two schools of 'same example across languages' and the 'rather show off R strength for modeling' never converged.

**Comment by karthik** [comment]
> Good to discuss in person

Right, that is the idea. Email discussions on such matters never lead anywhere (see SWC discuss list).

**Comment by mine-cetinkaya-rundel** [comment]
I'm certainly up for that discussion in person

**Comment by hadley** [comment]
:+1:

**Comment by cboettig** [comment]
:+1:

**Comment by rdpeng** [comment]
I am 100% down with having this discussion.

On Wed, Mar 4, 2015 at 10:08 AM, Tracy Teal notifications@github.com
wrote:

> At the unconference will be a lot of people involved in various teaching
> efforts in R. It seems like a great opportunity to get together to discuss
> what has been working most effectively and in different formats (workshops,
> courses, MOOCs) and where we might share or help contribute to each other's
> materials.
> 
> @jennybc https://github.com/jennybc @_inundata @eddelbuettel
> https://github.com/eddelbuettel @mine-cetinkaya-rundel
> https://github.com/mine-cetinkaya-rundel @richfitz
> https://github.com/richfitz @benmarwick https://github.com/benmarwick
> @rdpeng https://github.com/rdpeng @fmichonneau
> https://github.com/fmichonneau @amyehodge https://github.com/amyehodge
> @hadley https://github.com/hadley
> 
> For Data Carpentry, I'm interested in how can we can teach people new to R
> the big wins, and the things that they'll really use, without presenting
> things as too magical. We're also interested in content and perspectives on
> teaching how to work with APIs in R.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ropensci/unconf/issues/23.

## 

Roger D. Peng  |  @rdpeng https://twitter.com/rdpeng |
http://www.biostat.jhsph.edu/~rpeng/

**Comment by jennybc** [comment]
Count me IN.

**Comment by benmarwick** [comment]
Yep me too

**Comment by amyehodge** [comment]
Sounds like a great topic. 

~ Amy 

Amy E. Hodge, PhD 
Science Data Librarian 
Data Management Services 
Branner Earth Sciences Library, 212 Mitchell 
397 Panama Mall; MC 2211 
Stanford University 
Stanford, CA 94305 
amyhodge@stanford.edu 
650.556.5194 

----- Original Message -----

**Comment by karthik** [comment]
Glad to see so much support for this topic (I hoped so!). 
How about we set aside 2 hours for this discussion? If there is another discussion type topic, we could do that in a morning or afternoon session, perhaps on Friday.

**Comment by stephaniehicks** [comment]
+1, I'm in!

**Comment by tracykteal** [comment]
Thanks, the interest in this has been terrific! Looking forward to the discussion. It would be good for the discussion to have some structure, so maybe we could think about how to do that.

**Comment by tracykteal** [comment]
Thanks for the panel discussion @jennybc @mine-cetinkaya-rundel @hadley @benmarwick @rdpeng and all the good questions. Notes from the discussion are here https://ropensci.etherpad.mozilla.org/14 and we'll try to edit the video and get that posted.
